ui/edit_profile_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require_relative './spec_helper'
+
+describe 'Edit Profile' do
+  let(:dfe) { Dfe.new }
+
+  before do
+    dfe.home.load
+    dfe.home.header.sign_in.click
+
+    dfe.sign_in.wait_until_header_visible
```

Comment:
If the verbosity is better, I'd like to know what I should be using `Dfe` for further down the line? What do I get over `SignIn.new`

Reply:
@adeoke if you're unfamiliar with Yard gimme a shout it'd be great to get some documentation alongside your models.

Reply:
Great looking docs in the demo, make sure it builds HTML without errors. I mentioned about making the ENVs the default argument values in the method so it's even more convenient.

Reply:
For now, I'll leave out the password details as defaults. The reason being is that it makes the method a bit cryptic to the reader. unless I alias the method to make it less cryptic

current signature is:

with_email_and_password

alias could be

with_default_username_and_password
for example.

But again I think this can be omitted for the time being.